### PR TITLE
Fix exit codes

### PR DESCRIPTION
**Pull request by alysbrooks**
Fix some duplicated exit codes.

### CODE REVIEW

**Comment by plexus** [comment]
Nice! Thanks for picking this up. This will also need a CHANGELOG entry since it's technically a breaking change.

File: src/kaocha/config.clj [hunk 1]
```diff
@@ -29,7 +29,7 @@
         (update config k vary-meta assoc :replace true)
         (do
           (output/error "Test suite configuration value with key " k " should be a collection or symbol, but got '" v "' of type " (type v))
-          (throw+ {:kaocha/early-exit 252}))))
+         (throw+ {:kaocha/early-exit 250}))))
```
Comment (plexus) [inline src/kaocha/config.clj:32]: Careful with whitespace!

**Comment by alysbrooks** [comment]
@plexus Thanks, fixed! I was thinking about the `register` function, which has two slightly different errors (or three if you want to be even more precise) that all result in an exit code of 249. Do you think we should use separate codes for each of those? I was thinking we'd reserve the 240s (240-249 inclusive) for plugin registration errors, in case people want to catch all of them.

**Comment by plexus** [comment]
I'd have to look at the code, I probably am to blame her for trying to group related or similar errors, but while debugging an issue recently it was quite frustrating to not know which of two of three cases caused the error. This is especially true if the output (eg error message) is indistinguishable.

So the question to ask is: if something goes wrong, can we pinpoint exactly which file and line caused it. If not we need to differentiate.

**Comment by alysbrooks** [comment]
@plexus I separated failures to load namespaces from failures to load the plugin. The last two cases are separate paths but are only separate so we can use the correct grammar when there's one failed namespace versus multiple.

**Comment by plexus** [comment]
hat_off

**Pull request merged**